Autoturism.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Autoturism.h"
+#include <iostream>
+
+int Autoturism::numarObiecte = 0;
+
+Autoturism::Autoturism(const std::string combustibil, const std::string caroserie,const std::string tractiune, int putere,const std::string echipare, int* vin)
+	: Automobil( combustibil, caroserie, tractiune, putere, echipare, vin) {
+		if (numarObiecte >= 1){

```

Comment:
Aici nu avem o limitare in crearea obiectelor, difera doar mesajul printat de constructor, se pot crea oricate obiecte.

Reply:
am actualizat codul